### PR TITLE
Disable cuda_distributions_test and converter_nomigraph_test on Windows.

### PR DESCRIPTION
**Pull request by ezyang**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#25305 Disable cuda_distributions_test on Windows.**

See https://github.com/pytorch/pytorch/issues/25304 and https://github.com/pytorch/pytorch/issues/25312

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

Differential Revision: [D17092777](https://our.internmc.facebook.com/intern/diff/D17092777)

### CODE REVIEW

**Review by peterjc123 — APPROVED** [review]

**Review by izdeby — APPROVED** [review]

**Comment by peterjc123** [comment]
@ezyang We could use goto but it is still not so elegant.

**Comment by peterjc123** [comment]
The goto version could be like this:
```cmd
for /r "." %%a in (*.exe) do (
    call ::libtorch_check "%%~na" "%%~fa"
)

goto :eof

:libtorch_check
if "%~1" == "c10_metaprogramming_test" goto :eof
if "%~1" == "module_test" goto :eof
if "%~1" == "cuda_distributions_test" goto :eof

echo Running "%~2"
call "%~2"
if errorlevel 1 exit /b 1

goto :eof

```

File: .jenkins/pytorch/win-test-helpers/test_libtorch.bat [hunk 1]
```diff
@@ -6,17 +6,22 @@ test_api.exe --gtest_filter="-IntegrationTest.MNIST*"
 
 if errorlevel 1 exit /b 1
 
-cd %TMP_DIR_WIN%\build\torch\test
```
Comment (peterjc123) [inline .jenkins/pytorch/win-test-helpers/test_libtorch.bat]: This line is necessary. Why is it removed?

Comment (peterjc123) [reply .jenkins/pytorch/win-test-helpers/test_libtorch.bat]: Because the tests are skipped.

**Pull request closed without merging**